question28/js/password.js

```diff
@@ -29,6 +36,9 @@ for (const input of [passwordInputArea, confirmPasswordInputArea]) {
   });
 }
 
+passwordToggleButtons.forEach((passwordToggleButton) => {
+  passwordToggleButton.addEventListener('click', (e) => togglePassword (e.target)); 
```

Comment:
IMO
```suggestion
  passwordToggleButton.addEventListener('click', togglePassword); 
```
としてcallback関数がeから受け取る
特にこうかく理由がなさそうなので

Reply:
Q.
togglePassword()はイベントが起こった要素(e.target)を取得し引数に渡して動かしているので、ここは省けないと思いましたが、何か認識違いありそうでしょうか？？:eyes:

Reply:
書きました
https://github.com/csakyo/homework/pull/44#discussion_r1096644672